client/wildcard/src/components/Alert/Alert.story.tsx

```diff
@@ -0,0 +1,76 @@
+import { action } from '@storybook/addon-actions'
+import { Story, Meta } from '@storybook/react'
+import classNames from 'classnames'
+import { flow } from 'lodash'
+import React from 'react'
+
+import { BrandedStory } from '@sourcegraph/branded/src/components/BrandedStory'
+import webStyles from '@sourcegraph/web/src/SourcegraphWebApp.scss'
+
+import 'storybook-addon-designs'
+
+import { Alert } from './Alert'
+import styles from './Alert.module.scss'
+import { ALERT_VARIANTS } from './constants'
+
+import { AlertLink } from '.'
+
+const preventDefault = <E extends React.SyntheticEvent>(event: E): E => {
+    event.preventDefault()
+    return event
+}
+
+const config: Meta = {
+    title: 'wildcard/Alert',
+    decorators: [
+        story => (
+            <BrandedStory styles={webStyles}>{() => <div className="container mt-3">{story()}</div>}</BrandedStory>
+        ),
+    ],
+    parameters: {
+        component: Alert,
+        design: [
+            {
+                type: 'figma',
+                name: 'Figma Light',
+                url:
+                    'https://www.figma.com/file/NIsN34NH7lPu04olBzddTw/Design-Refresh-Systemization-source-of-truth?node-id=1563%3A196',
+            },
+            {
+                type: 'figma',
+                name: 'Figma Dark',
+                url:
+                    'https://www.figma.com/file/NIsN34NH7lPu04olBzddTw/Design-Refresh-Systemization-source-of-truth?node-id=1563%3A525',
+            },
+        ],
+    },
+}
+
+export default config
+
+export const Alerts: Story = () => (
+    <>
+        <h1>Alerts</h1>
+        <p>
+            Provide contextual feedback messages for typical user actions with the handful of available and flexible
+            alert messages.
+        </p>
+        <div>
+            {ALERT_VARIANTS.map(variant => (
+                <Alert key={variant} variant={variant}>
+                    <h4>Too many matching repositories</h4>
+                    Use a 'repo:' or 'repogroup:' filter to narrow your search.
+                </Alert>
+            ))}
+            <Alert className={classNames('d-flex align-items-center', styles.alert, styles.alertInfo)}>
```

Comment:
It seems that here styles are redundant because they are already provided by the `<Alert />` component

```suggestion
            <Alert variant="info" className="d-flex align-items-center">
```